### PR TITLE
Dropdown lowercase label and responsivity

### PR DESCRIPTION
**Pull request by jordanmcousins**

### CODE REVIEW

File: apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader'
-import axios from "axios";
+import React from 'react';
+import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader';
+import axios from 'axios';
 import { atoms } from '@devlaunchers/components/src/components';
 import IdeaCard from '../../common/IdeaCard/IdeaCard';
 import BackButton from '../../common/BackButton/BackButton';
 import Dropdown from '@devlaunchers/components/components/organisms/Dropdown';
+import useResponsive from '@devlaunchers/components/src/hooks/useResponsive';
```
Comment (Enjoy2Live) [inline apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js:8]: I see that you guys are using the hooks we provided without us even telling about them, I hope it works well and if not please always create issues with the label platform enablement :D

Comment (jordanmcousins) [reply apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js:8]: Thanks Mohammed haha that was a recommendation from Dave, I'm not sure how he found out that hook existed

Comment (dbradham) [reply apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js:8]: I learned it from Anwar (:

**Review by dbradham — APPROVED** [review]
Looks good!

**Pull request merged**